### ISSUE TITLE
google-cloud-bom 0.120.0 has incorrect group IDs for a number of dependencies

### ISSUE THREAD

**Issue by elharo** [googleapis/google-cloud-java#6996]
[ERROR] testMaximumLinkageErrors(com.google.cloud.MaximumLinkageErrorsTest)  Time elapsed: 79.13 s  <<< ERROR!
org.eclipse.aether.resolution.DependencyResolutionException: The following artifacts could not be resolved: com.google.cloud:grpc-google-cloud-error-reporting-v1beta1:jar:0.84.0, com.google.cloud:proto-google-cloud-error-reporting-v1beta1:jar:0.84.0, com.google.cloud:google-cloud-kms-bom:jar:1.35.0, com.google.cloud:google-cloud-video-intelligence-bom:jar:0.116.0-beta: Could not find artifact com.google.cloud:grpc-google-cloud-error-reporting-v1beta1:jar:0.84.0 in central (http://repo1.maven.org/maven2/)

Still investigating, could be a faulty assumption in this test, but it looks like something is pointing to com.google.cloud:grpc-google-cloud-error-reporting-v1beta1:jar:0.84.0 when it should point to com.google.api.grpc:grpc-google-cloud-error-reporting-v1beta1:0.84.0

Error appears to be here: https://repo1.maven.org/maven2/com/google/cloud/google-cloud-errorreporting-bom/0.118.0-beta/google-cloud-errorreporting-bom-0.118.0-beta.pom
(and likely in the other artifacts too)
Here's the bug:
```xml
<dependencyManagement>
    <dependencies>
      <dependency>
        <groupId>com.google.cloud</groupId>
        <artifactId>google-cloud-errorreporting</artifactId>
        <version>0.118.0-beta</version><!-- {x-version-update:google-cloud-errorreporting:current} -->
      </dependency>
      <dependency>
        <groupId>com.google.cloud</groupId>
        <artifactId>grpc-google-cloud-error-reporting-v1beta1</artifactId>
        <version>0.84.0</version><!-- {x-version-update:grpc-google-cloud-errorreporting-v1beta1:current} -->
      </dependency>
      <dependency>
        <groupId>com.google.cloud</groupId>
        <artifactId>proto-google-cloud-error-reporting-v1beta1</artifactId>
        <version>0.84.0</version><!-- {x-version-update:proto-google-cloud-errorreporting-v1beta1:current} -->
      </dependency>
    </dependencies>
  </dependencyManagement>
```

Group IDs are incorrect

source repo looks correct though. Could this be a bug in the release process?

**Comment by elharo** [comment]
0.82.0 from 10/31 had the correct group IDs. The error was pushed yesterday.

**Comment by codyoss** [comment]
This should be fixed as of googleapis/java-cloud-bom#136. Closing.
